B827EBFFFE430F6B.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE430F6B", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-asia-se", 
```

Comment:
This is not a valid server address, check the valid ones in this list: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses